index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Natnael Front-End Developer</title>
+</head>
+<body>
+    <div class="first">
+        <header class="header bg-white">
+          <nav class="navbar flex">
+            <a href="#" class="navbar__link ">NA</a>
+            <div class="navbar__menu">
+              <ul class="navbar__menu__ul ul flex">
+                <li class="navbar__menu__li">
+                  <a href="#Portfolio" class="navbar__menu__link">Portfolio</a>
+                </li>
+                <li class="navbar__menu__li">
+                  <a href="#about" class="navbar__menu__link link">About</a>
+                </li>
+                <li class="navbar__menu__li">
+                  <a href="#contact" class="navbar__menu__link link">Contact</a>
+                </li>
+              </ul>
+            </div>
+            <img class="navbar__list" src="images/list.png" alt="list icon" />
+          </nav>
+        </header> 
+        <section class="head flex">
+            <h1 class="head__title">
+              I'm Natnael <span>Glad to see you!</span>
+            </h1>
+            <p class="head__con">
+              I’m a software developer! I can help you build a product , feature or
+              website Look through some of my work and experience! If you like what
+              you see and have a project you need coded, don’t hestiate to contact
+              me.
+            </p>
+            <a href="#works" class="head__link">LET'S CONNECT!</a>
```

Comment:
- [ ] Please change this anchor tag to the `<hx>` heading like h2, h3, h4... and use the correct color and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165493822-850eddd4-4755-48fa-8b59-60090bcadc7b.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165493863-8837c6f1-3ebe-4bbf-8a50-35593d0be00a.png)